packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -1,12 +1,87 @@
 // @flow
 import * as React from "react";
 
-type Props = {||};
+import {Text as WbText} from "@khanacademy/wonder-blocks-core";
 
-type State = {||};
+import TooltipPortalMounter from "./tooltip-portal-mounter";
+import TooltipAnchor from "./tooltip-anchor.js";
+import TooltipBubble from "./tooltip-bubble.js";
+
+import type {Placement} from "../util/types.js";
+
+type Props = {|
+    // The content for anchoring the tooltip.
+    // This component will be used to position the tooltip.
+    children: React.Element<any> | string,
+
+    // The content to render in the tooltip.
+    // TODO(somewhatabstract): Update to allow TooltipContent or string
+    content: string,
+
+    // When true, the child element will be given tabindex=0
+    // to make it keyboard focusable. This value defaults to true. One might set
+    // this to false in circumstances where the wrapped component already can
+    // receive focus or contains an element that can.
+    // Use good judgement when overriding this value, the tooltip content should
+    // be accessible via keyboard in all circumstances where the tooltip would
+    // appear using the mouse, so very those use-cases.
+    forceAnchorFocusivity?: boolean,
```

Comment:
I think we change this, but in a separate PR since there are other files involved.   Maybe `isAnchorFocusable`?

Reply:
I don't want to give the illusion that we're turning focus off ever.

How about `forceAnchorToBeFocusable`?

Reply:
Ugh... let's leave it for now.